### ISSUE TITLE
Op run does not preserve color

### ISSUE THREAD

**Issue by ShravanSunder** [1Password/shell-plugins#413]
### op CLI version

2.22

### Goal or desired behavior

I'm running op run with wrangler
op run --env-file='.dev.vars' -- wrangler dev src/worker.ts --port 8888

The colors from wrangler and the formatting of its cli is not preserved

### Current behavior

The cli removes color and formatting.  Its hard to parse debug information and makes it hard to use 1p cli

**Comment by damccull** [comment]
Also experiencing this, but with rust's cargo run and other commands.

**Comment by digitalcora** [comment]
This issue appears to be tied to the "masking" feature, since I get color output using `op run --no-masking` (or when there are no `op://` references in the environment, which I guess disables the feature automatically). Of course, if you use this as a workaround, you don't get the benefit of masking.

**Comment by ShravanSunder** [comment]
@florisvdg sorry to ping you, i wanted to reach out to someone in 1p team.  

Is it possible for someone from 1p to look at this?  It makes it hard to use op cli to protect secrets when it changes the behaviour of the commands we need to invoke.

**Comment by choccccy** [comment]
> This issue appears to be tied to the "masking" feature, since I get color output using `op run --no-masking` (or when there are no `op://` references in the environment, which I guess disables the feature automatically). Of course, if you use this as a workaround, you don't get the benefit of masking.

I also run into this issue, and find that using the `--no-masking` arg fixes it.

**Comment by ubahmapk** [comment]
+1

**Comment by sckott** [comment]
I'm finding `op run` unusable when trying to use a command after it. For example, `op run -- python` completely removes any coloring and disables ctrl+r searching history and keying up for history. Works fine if you add `--no-masking` but ideally secrets are hidden. Pretty please fix

**Comment by nikclayton** [comment]
It's not just colour codes. Codes that move the cursor are also stripped.

For example, I have some automation that has to poll a remote system at one point in the process. Without `op run` this prints:

```
Health check 1/10
```

Then a bit later it moves the cursor up and overwrites that line to print

```
Health check 2/10
```

and so on.

With `op run` the cursor control codes are ignored, and this prints as multiple lines:

```
Health check 1/10
Health check 2/10
```

etc.

As with the colour issue, passing `--no-masking` resolves this.

**Comment by nikclayton** [comment]
Also, just noticed that if I'm running a command that hides its output (e.g., gpg prompting for a passphrase) the hidden output is printed if run using "op run".
